release.nix

```diff
@@ -4,12 +4,10 @@
 }:
 
 let
-  inherit (release) mapTestOn packagePlatforms pkgs all linux darwin;
+  inherit (release) mapTestOn packagePlatforms pkgs;
 
   system = "x86_64-darwin";
```

Comment:
```suggestion
```

If you have the time or energy, would you be interested in removing the hardcoded `system` and running the tests on `aarch64-darwin` as well?

Reply:
I can only run things on x86-64_linux and aarch64-linux 😅